src/cascadia/TerminalSettingsModel/ColorScheme.h

```diff
@@ -47,11 +47,13 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         com_array<Windows::UI::Color> Table() const noexcept;
         void SetColorTableEntry(uint8_t index, const winrt::Windows::UI::Color& value) noexcept;
 
+        WINRT_CALLBACK(PropertyChanged, Windows::UI::Xaml::Data::PropertyChangedEventHandler);
+
         GETSET_PROPERTY(winrt::hstring, Name);
-        GETSET_COLORPROPERTY(Foreground); // defined in constructor
-        GETSET_COLORPROPERTY(Background); // defined in constructor
-        GETSET_COLORPROPERTY(SelectionBackground); // defined in constructor
-        GETSET_COLORPROPERTY(CursorColor); // defined in constructor
+        OBSERVABLE_GETSET_COLORPROPERTY(Foreground, _PropertyChangedHandlers); // defined in constructor

```

Comment:
concerned abt making these observable -- why did it become required, and will it complicate Profile and the other pages?

Reply:
I made them observable because in the `ColorSchemes` page, each of these colors are bound to multiple places (e.g. a ColorPicker and a TextBox), and so changing the color in one of the bindings needs to notify the other binding that the color has changed.